### PR TITLE
Kw-feature mycampaignslist

### PR DESCRIPTION
**Pull request by KanikaWadhawan**
# Feature MyCampaigns Screen

- Fetch Data from Database to display all the subscribed campaigns.
- Style the MyCampaigns Screen
- Make the Unsubscribe button to unsubscribe from the Campaign
- On Unsubcribing, the Campaign disappeares from the MyCampaigns Screen.

### CODE REVIEW

File: client/js/components/SmallUnsubscribeButton/SmallUnsubscribeButton.js [hunk 1]
```diff
@@ -1,12 +1,54 @@
...
-      <Text style={styles.buttonText}>Unsubscribe</Text>
-    </TouchableOpacity>
+    <Mutation mutation={CAMPAIGN_UNSUBSCRIBE} client={client}>
+      {updateUser => {
+        return (
+          <View
+            style={{
+              height: '70%',
+              width: '24%',
+            }}>
```
Comment (siddharthparmar7) [inline client/js/components/SmallUnsubscribeButton/SmallUnsubscribeButton.js]: Please move this styles to styles.js file

File: client/js/components/SmallUnsubscribeButton/SmallUnsubscribeButton.js [hunk 2]
```diff
@@ -1,12 +1,54 @@
...
+            <TouchableOpacity
+              onPress={async () => {
+                try {
+                  const editSubscribe = await updateUser({
+                    variables: {
+                      campaignid: CampaignId,
+                      userid: userid,
+                    },
+                  });
+                } catch (e) {}
```
Comment (siddharthparmar7) [inline client/js/components/SmallUnsubscribeButton/SmallUnsubscribeButton.js]: what should we do when there is an error?

File: client/js/screens/MyCampaigns/MyCampaigns.js [hunk 3]
```diff
@@ -12,62 +12,44 @@ import styles from './styles';
...
+                    flexDirection: 'row',
+                    justifyContent: 'space-between',
+                    height: '100%',
+                  }}
+                  onPress={() => {
+                    navigation.navigate('Campaign', {
+                      campaign: item,
+                    });
+                  }}>
+                  {/* View for text flexDirection = column (auto)*/}
```
Comment (siddharthparmar7) [inline client/js/screens/MyCampaigns/MyCampaigns.js]: Please move all the inline styles to styles.js file and remove the commented code if not needed.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Nice work. Please make the changes.

File: client/js/screens/MyCampaigns/MyCampaigns.js (hunk 3 above)
Comment (KanikaWadhawan) [reply client/js/screens/MyCampaigns/MyCampaigns.js]: Done.

File: client/js/components/SmallUnsubscribeButton/SmallUnsubscribeButton.js (hunk 2 above)
Comment (KanikaWadhawan) [reply client/js/components/SmallUnsubscribeButton/SmallUnsubscribeButton.js]: Done

File: client/js/components/SmallUnsubscribeButton/SmallUnsubscribeButton.js (hunk 1 above)
Comment (KanikaWadhawan) [reply client/js/components/SmallUnsubscribeButton/SmallUnsubscribeButton.js]: Done

**Review by siddharthparmar7 — APPROVED** [review]

**Pull request merged**